### PR TITLE
gptfdisk 1.0.7

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
clang: error: no such file or directory: '/usr/local/Cellar/ncurses/6.2/lib/libncurses.dylib'
make: *** [cgdisk] Error 1
```

**Comment by carlocab** [comment]
Looks like they switched from looking for `/usr/lib/libncurses.dylib` to looking for Homebrew `libncurses.dylib`. Except they also hardcoded the prefix.

File: Formula/gptfdisk.rb [hunk 1]
```diff
@@ -38,15 +36,15 @@ def install
...
+index ea21fa6..b50bb34 100644
+--- a/Makefile.mac
++++ b/Makefile.mac
+@@ -21,7 +21,7 @@ gdisk:	$(LIB_OBJS) gpttext.o gdisk.o
  #	$(CXX) $(LIB_OBJS) -L/usr/lib -licucore gpttext.o gdisk.o -o gdisk
  
  cgdisk: $(LIB_OBJS) cgdisk.o gptcurses.o
--	$(CXX) $(LIB_OBJS) cgdisk.o gptcurses.o /usr/lib/libncurses.dylib $(LDFLAGS) $(FATBINFLAGS) -o cgdisk
-+	$(CXX) $(LIB_OBJS) cgdisk.o gptcurses.o -L/usr/lib -lncurses $(LDFLAGS) $(FATBINFLAGS) -o cgdisk
+-	$(CXX) $(LIB_OBJS) cgdisk.o gptcurses.o /usr/local/Cellar/ncurses/6.2/lib/libncurses.dylib $(LDFLAGS) -o cgdisk
```
Comment (SMillerDev) [inline Formula/gptfdisk.rb:47]: Can't this link to the `opt_prefix`?

Comment (cho-m) [reply Formula/gptfdisk.rb:47]: Both the previous diff (used in v1.0.6) and new diff here (v1.0.7) are patching Makefile to use macOS `ncurses`, corresponding to the `uses_from_macos "ncurses"`

In this case, the diff is doing:
`/usr/local/Cellar/ncurses/6.2/lib/libncurses.dylib` ==> `-L/usr/lib -lncurses`

The alternative would be to use Homebrew's `ncurses` along with corresponding path.

Comment (SMillerDev) [reply Formula/gptfdisk.rb:47]: Damned, misread the patch. `+ -` at the start of the line is confusing

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**